blaze/compute/sql.py

```diff
@@ -789,3 +792,37 @@ def post_compute(_, s, **kwargs):
 @dispatch(IsIn, ColumnElement)
 def compute_up(expr, data, **kwargs):
     return data.in_(expr._keys)
+
+
+@dispatch(Slice, ClauseElement)
```

Comment:
I like the idea of a warning when this is triggered. We should let people know what kind of assumption they're making when they choose to use `Slice` expressions on a relational database table. Something like:

```
The order of the result set from a Slice expression computed with the SQL backend is not deterministic.
```